packages/e/e2fsprogs/xmake.lua

```diff
@@ -0,0 +1,28 @@
+package("e2fsprogs")
+
+    set_homepage("http://e2fsprogs.sourceforge.net")
+    set_description("Filesystem utilities for the ext2/3/4 filesystems")
+
+    add_urls("https://git.kernel.org/pub/scm/fs/ext2/e2fsprogs.git/snapshot/e2fsprogs-$(version).tar.gz")
+
+    add_versions("1.46.4", "c011bf3bf4ae5efe9fa2b0e9b0da0c14ef4b79c6143c1ae6d9f027931ec7abe1")
+
+    if is_plat("linux") then
+        add_extsources("apt::e2fsprogs")
+    elseif is_plat("macosx") then
+        add_extsources("brew::e2fsprogs")
+    end
+
+    on_install(function (package)
```

Comment:
ios, windows ci fails. you can just add supported platforms here.

for example:

```
on_install("linux", "macosx", function(package)
end
```

Reply:
Done